### DISCUSSION TITLE
Community brand

### DISCUSSION THREAD

**Discussion by AlexxNica** [memser-spaceport/pl-network-hub#8] [Ideas]
### What is the idea?

It'd be nice to have a brand for the Protocol Labs Network community to use in public/open-source materials, such as having a `Made with ❤️ at the PLN` or `Made with ❤️ as part of PLN`, as suggested by @bmann.

### Why is this important?

Increase brand awareness through community contribution and creativity, which drives engagement and growth.

**Comment by bmann** [comment]
Thanks @AlexxNica — yep, would be nice to have little touches like this. And ideally the PLN links all go to https://plnetwork.io as the home base.

**Reply by pandeyanujk** [reply]
@bmann [here](https://www.figma.com/file/4eksDQB0KH4dGJmhrvv96u/PLN-Social-Media-%26-Community-requests?node-id=4%3A15&t=rCJYaRkKFEXrxKiL-1) you go.

**Comment by pandeyanujk** [comment]
Closing this request since this has been delivered.
